packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -3469,6 +3555,29 @@ export interface IDetailsRowCheckStyles {
     root: IStyle;
 }
 
+// @public (undocumented)

```

Comment:
Not a blocker 😊 ... but possible to document everything we export as public API? We should at least have some short description so that we don't add more of these `// @public (undocumented)` tags.

Reply:
Sure..

Reply:
Done in 83c55e9da85dfe2d8ce1dd310a46b598101baaf2..74a62991ff73a1c2b88596108ab46936b6953aea.